docs/week-5/robot-code.md

```diff
@@ -95,14 +95,14 @@ intakeBtn.whenPressed(new RunIntake());
+Subsystems are another name for robot mechanisms, such as a drivetrain, an elevator, an intake, or a climber. Each subystem class contains methods that control a subsystem using the objects created by the `RobotMap` for the motor controllers, solenoids, and sensors that are part of that subsystem.
 
 ## Commands
-Commands are what run the robot. For example, there may be a `RunIntake` command that runs two motors to the wheels that intake a game piece. 
+Commands use subsystem methods to control what happens at the beginning of an action, during an action, and at the end of an action, as well as to determine  when an action should stop. For example, there may be a `RunIntake` command that starts the intake wheels spinning to grab a game piece and stops them when a an intkae sensor determines that the game piece has been grabbed. Once a command has been started, it runs periodically until it is interrupted or it determines that it has finished.
 
-In addition there are instant commands, which run instantly. For example, you might have an instant command called `ActuatePiston` that actuates a piston on the robot.
+Instant commands are a special kind of command that does not need to run periodically after it is started. It does something and then is considered done. For example, you might have an instant command called `ActuatePiston` that actuates a piston on the robot. Since the piston will automatically stay in its new position, there is no need to periodically actuate the piston to that piston.
 
-Finally, there are command groups. Command groups run a series of commands. You can mix and match whether you want to run a command right after the previous one (called running sequentially) or at the same time as the previos command (called running in parallel). 
+Finally, there are command groups. Command groups run a series of commands. You can mix and match whether you want to run commands one after the other (called running sequentially) or at the same time (called running in parallel). 
 
 ## Lib
-The `lib` folder is where we would store the complex calculations, wrappers, or other things that do not directly interact with the robot, but are rather called on by commands or subsystems to assist in their execution. Stuff in this folder would be valuable code that we would actually want to keep and develop on from one year to another, such as drivetrain characterization, motion profiling, or swerve drive code.
\ No newline at end of file
+The `lib` folder is where we would store the complex calculations, wrappers, or other things that do not directly interact with the robot, but are rather called on by commands or subsystems to assist in their execution. Stuff in this folder is code that we might actually want to keep and develop from one year to another, such as drivetrain characterization, motion profiling, or swerve drive code.
```

Comment:
i wrote it the way i did cuz we don't yet have anything that is put into a lib folder yet 😆. (If we want to use Lex's characterization code in future robot codes we'll need to change it up a bit to be more code-agnostic)

Reply:
I don't understand the concern with my proposed change. I added "is" as a grammar correction and added "might" because we won't necessarily reuse code in the lib folder in a future year, but code which is a candidate for such reuse should go there.